### PR TITLE
 #158052290 Refactor unit-tests for endpoints

### PR DESCRIPTION
**Pull request by joelmachango**
**What does this PR do?**
Update the test cases the project.

**Description of Task to be completed**
1. Create unit tests for endpoints.
2. Update directory structure
3. Improve existing tests

| Endpoint                                                | Functionality         | HTTP method |
| ------------------------------------------------------- | --------------------- | ----------- |
| /requests/api/v1/requests              | Fetch all requests    | GET         |
| /requests/api/v1/requests[request_id]  | Fetch a requests      | GET         |
| /requests/api/v1/requests              | Create a requests     | POST        |
| /requests/api/v1/requests[request_id]  | Modify a requests     | PUT         |

**How should this be manually tested?**
Clone the repository
`$ git clone https://github.com/joelmachango/maintenance-tracker`

`$git checkout ch-ch-refactor-endpoint-unittests-#158052290`

cd maintenance-tracker/app/

`$ python3 -m venv venv`

`$. venv/bin/activate`

`$ pip install -r requirements.txt

  $ pytest`

**Any background context you want to provide?**
This PR consist of unit tests for the app

**What are the relevant pivotal tracker stories?**
[#158052290](https://www.pivotaltracker.com/n/projects/2173255/stories/158052290)

### CODE REVIEW

File: app/models/users.py [hunk 1]
```diff
@@ -16,9 +16,9 @@ def register(self, username, password, confirmfpassword):
             if username == user['username']:
                 return "Sorry, Username exists."               
         else:
-            #Check if Username and Password format is correct.
-            if not re.match("^[a-zA-Z0-9_]*$", username):
-                return "Please use Alphanumeric characters"
+            #Check if Username and Password format is correct.
```
Comment (artorious) [inline app/models/users.py]: regex.........Awesome!!

**Comment by artorious** [comment]
Looks/Runs Good Bruv

**Comment by maxwellgithinji** [comment]
Looks good.

**Comment by Slinjez** [comment]
The PR is nicely structured

**Pull request merged**